weaviate/collections/classes/aggregate.py

```diff
@@ -134,6 +136,12 @@ class _MetricsBase(BaseModel):
     property_name: str
     count: bool
 
+    def to_gql(self) -> str:
+        raise NotImplementedError
+
+    def to_grpc(self) -> aggregate_pb2.AggregateRequest.Aggregation:
+        raise NotImplementedError
```

Comment:
no need to change this now, but wouldn't an abstract baseclass be better that adding the NotImplemented?

Reply:
If it was labelled as an abstract base class, wouldn't it still need the `NotImplementedError` thrown to indicate that children were not implemented correctly at runtime? Or does `mypy` catch these sorts of issues?

Reply:
You cannot instantiate abstract classes, so you don't need it (IIRC, has been a while)

Reply:
But you could inherit from the abstract base class and forget to implement `_to_grpc()`. This error then throws in such a case